### DISCUSSION TITLE
CKEditor (custom rich text editor) not working in production

### DISCUSSION THREAD

**Discussion by hoangkimminh** [strapi/strapi#7483] [Legacy Topics]
I customized Strapi Rich text editor with CKEditor 5 by using tutorial: [strapi.io/blog/how-to-change-the-wysiwyg-in-strapi](strapi.io/blog/how-to-change-the-wysiwyg-in-strapi).
It had worked before recent build, I didn't change anything in Editor at that build.
Does anyone have same isssue?
May you suggest me some solutions, please?

**Comment by lauriejim** [comment]
Ping @soupette for this one.
But I think this guide is updated rather that the article you linked
https://strapi.io/documentation/v3.x/guides/registering-a-field-in-admin.html

**Reply by hoangkimminh** [reply]
Thanks @lauriejim. I've tried that tutorial but it had same problem. I think dependencies in wysiwyg folder is missing so I change Dockerfile to this:
`# Use the official lightweight Node.js 12 image.
# https://hub.docker.com/_/node
FROM node:12-slim

# Create and change to the app directory.
WORKDIR /usr/src/app

# Copy application dependency manifests to the container image.
# A wildcard is used to ensure both package.json AND package-lock.json are copied.
# Copying this separately prevents re-running npm install on every code change.
COPY package*.json ./

# Install wysiwyg dependencies
WORKDIR /usr/src/app/plugins/wysiwyg
RUN npm install

# Install production dependencies.
WORKDIR /usr/src/app
RUN npm install

RUN NODE_ENV=development npm run-script build

# Copy local code to the container image.
COPY . ./

# Run the web service on container startup.
CMD [ "npm", "start" ]`
But it does'nt change

**Comment by adrian-ub** [comment]
You need install dependencies in plugin folder

Ex: 
```
cd plugins/wysiwyg  
yarn / npm install
```

**Reply by hoangkimminh** [reply]
@adrian-ub I've tried that but default editor've displayed in production
https://github.com/strapi/strapi/discussions/7483#discussioncomment-59109
